Tests/BasicsTests/TemporaryFileTests.swift

```diff
@@ -9,11 +9,7 @@
  */
 
 import XCTest
-#if swift(>=5.7)
```

Comment:
should we change SwiftPM manifest to tools-version 5.7 and get rid of all these?

Reply:
Don't think we can, Swift CI is still using 5.6 for some jobs.